### PR TITLE
Feature/add layers impl for category class 0005/master

### PR DESCRIPTION
**Pull request by MackowiakPiotr**

### CODE REVIEW

File: src/main/java/com/baal/WebShop/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Validated
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("admin/category")
```
Comment (lukmoneta) [inline src/main/java/com/baal/WebShop/controller/CategoryController.java:19]: wrong value for mapping

File: src/main/java/com/baal/WebShop/mapper/CategoryModelMapper.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.baal.WebShop.mapper;
+
+import com.baal.WebShop.DTO.CategoryDTO;
+import com.baal.WebShop.model.Category;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface CategoryModelMapper {
+    CategoryDTO mapCategoryEntityToCategoryDTO(Category category);
```
Comment (lukmoneta) [inline src/main/java/com/baal/WebShop/mapper/CategoryModelMapper.java:9]: I think here you should explicit add methods to map the whole chain of classes, but if application is running as intended please do nothing here.

File: src/main/java/com/baal/WebShop/controller/CategoryController.java (hunk 1 above)
Comment (MackowiakPiotr) [reply src/main/java/com/baal/WebShop/controller/CategoryController.java:19]: What is the correct value?

File: src/main/java/com/baal/WebShop/mapper/CategoryModelMapper.java (hunk 2 above)
Comment (MackowiakPiotr) [reply src/main/java/com/baal/WebShop/mapper/CategoryModelMapper.java:9]: Work fine for me :)

**Review by WieckowskiMarek — APPROVED** [review]

**Review by WieckowskiMarek — APPROVED** [review]

**Review by lukmoneta — APPROVED** [review]

**Pull request merged**